client/src/components/Workflow/Run/WorkflowRunGraph.vue

```diff
@@ -66,6 +67,7 @@ try {
                 readonly />
         </BCard>
     </div>
+    <BAlert v-else variant="danger" show> Unable to load graph due to missing datatypes mapper. </BAlert>
```

Comment:
Is this actually reachable ? If it is there must be an actual error message somewhere.

Reply:
This somehow happened only in the case of workflow landings and trying to load the graph then. Let me take another look and try to find where it originates...